sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/persist/fileSystemPersist.ts

```diff
@@ -177,6 +179,7 @@ export class FileSystemPersist implements PersistentStorage {
         if (files.length === 0) {
           return false;
         } else {
+          // eslint-disable-next-line @typescript-eslint/no-misused-promises
           files.forEach(async (file) => {
```

Comment:
interesting, `forEach` should only work with sync functions. It seems like it is deleting files and perhaps there is no need to block until deletion finishes?